nengo_gui/examples/tutorial/25-spa-parse.py.cfg

```diff
@@ -1,50 +1,48 @@
-_viz_0 = nengo_gui.components.PointerTemplate(model.vision)
-_viz_config[_viz_0].label_visible = 1
+_viz_0 = nengo_gui.components.Pointer(model.vision,target='default')
```

Comment:
I like this new layout in general, but the ```phrase``` cloud seems placed on the opposite side from the ```phrase``` network, and it'd be good to have the BG input and the thalamus.actions.output plots.